api/setup.ts

```diff
@@ -0,0 +1,65 @@
+//Imports
+import { parse,  stringify } from "https://deno.land/std@0.97.0/encoding/yaml.ts";
+import { DOMParser, Element } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
+
+//Load static/base.html
+let html: any = new DOMParser().parseFromString(await Deno.readTextFile("./static/base.html"), "text/html")!
+
+//Load config.yml
+let config: any = parse(await Deno.readTextFile("./config.yml"))
+
+//Set company name
+if(config["name"]){
+    //TODO
+}
+
+//Set custom css
+config["styles"]!.forEach((stylePath: string ) => {
+    let attrs = {
+        "href" : "static/" + stylePath,
```

Comment:
```suggestion
        "href" : `static/${stylePath}`,
```
Both are working, but I find [template literals](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Template_literals) more sexy since they make it clear that you're willing to concatenate strings. I prefer using restricting the `+` operator for numbers